src/apps/cmake_install.cmake

```diff
@@ -0,0 +1,44 @@
+# Install script for directory: /home/wg25r/new/CMT/src/apps
```

Comment:
Files cmake_install.cmake is another (like Makefiles) that are auto generated and we dont want to install in
the repo. If you are adding them on purpose, let me know. But in the mean time I am adding them to the .gitignore.